src/parser/parser_utils.js

```diff
@@ -228,6 +228,10 @@ function mergeWrapperAdData(unwrappedAd, wrapper) {
   );
   unwrappedAd.extensions = wrapper.extensions.concat(unwrappedAd.extensions);
 
+  unwrappedAd.followAdditionalWrappers = wrapper.followAdditionalWrappers;
+  unwrappedAd.allowMultipleAds = wrapper.allowMultipleAds;
+  unwrappedAd.fallbackOnNoAd = wrapper.fallbackOnNoAd;
```

Comment:
we are essentially overriding the `unwrappedAd` values with the values of the parent wrapper. I don't think the IAB specs specify what to do in these cases, so at the very least we could leave a comment here that the values from the child wrapper will be overridden. @rumesh what do you think?

Reply:
sounds good for me